.github/workflows/ext-build-subtopology-ac.yml

```diff
@@ -0,0 +1,87 @@
+# Runs the subtopology autocoder tool to test the subtopology tool (https://github.com/mosa11aei/fprime-subtopology-tool)
+
+name: "Subtopology Autocoder Tool Test"
+
+on:
+  push:
+    branches: [ devel, release/** ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ devel, release/** ]
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+      - '.github/actions/spelling/**'
+      - '.github/ISSUE_TEMPLATE/**'
+
+jobs:
+  get-branch:
+    name: "Get target branch"
+    uses: ./.github/workflows/reusable-get-pr-branch.yml
+    with:
+      target_repository: mosa11aei/fprime-rngLibrary
+
+  subtopology-test:
+    name: "Subtopology Test"
+    runs-on: ubuntu-latest
+    needs: get-branch
+    steps:
+      - name: "Checkout target repository"
+        uses: actions/checkout@v4
+        with:
+          submodules: false
+          repository: mosa11aei/fprime-rngLibrary
+          ref: ${{ needs.get-branch.outputs.target-branch }}
+      - name: "Overlay current F´ revision"
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          path: ./fprime
+          fetch-depth: 0
+      - uses: ./fprime/.github/actions/setup
+        with:
+          location: ./fprime
+      - name: "Verify CMake is installed"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake
+      - name: "Get FPP"
+        uses: actions/checkout@v4
+        with:
+          submodules: false
+          repository: nasa/fpp
+          path: ./fpp
+          ref: main
+      - name: "[fpp] Install sbt"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y apt-transport-https curl gnupg -yqq
+          echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
+          echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
+          curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo -H gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/scalasbt-release.gpg --import
+          sudo chmod 644 /etc/apt/trusted.gpg.d/scalasbt-release.gpg
```

Comment:
Does it not work to leave all that out and run `apt-get install sbt` directly?

Reply:
Install instructions came from the sbt website:

https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html#Ubuntu+and+other+Debian-based+distributions

Reply:
It actually looks like SBT might be installed on the runners already
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Not sure of the version requirements with FPP though

Reply:
It should be sufficient!